hackathon_site/dashboard/frontend/src/api/helpers.ts

```diff
@@ -61,6 +61,12 @@ export const teamOrderListSerialization = (
                     hardwareInOrder: returnedHardware,
                 });
             const hardwareInTableRow = Object.values(hardwareItems);
+            const formattedCreatedDate = order.created_at.slice(0, 10);
```

Comment:
Looks good. Does the slice(0, 10) work always for any Date type?

Reply:
It should I believe. I think the Date object being retrieved (from Django?) is a JS datetime object right? I just referenced this website (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) which under the heading "Date Time String Format" lists how the date types are usually formatted